qa/test_template.sh

```diff
@@ -12,6 +12,12 @@ source $topdir/qa/setup_test_common.sh
 pip_packages=$(echo ${pip_packages} | sed "s/##CUDA_VERSION##/${CUDA_VERSION}/")
 last_config_index=$($topdir/qa/setup_packages.py -n -u $pip_packages --cuda ${CUDA_VERSION})
 
+install_pip_pkg() {
+    install_cmd=$1
```

Comment:
just a thought. Maybe instead of grabbing the first argument you could grab all arguments. Otherwise running this without the double quotes will not work:
```
install_pip_pkg pip install --force-reinstall horovod==0.21.3 -f /pip-packages
```
now
```
$1=pip
```

Reply:
Done